test/embedded/throw-typed.swift

```diff
@@ -0,0 +1,95 @@
+// RUN: %target-run-simple-swift(%S/Inputs/print.swift -enable-experimental-feature Embedded -enable-experimental-feature TypedThrows -parse-as-library -runtime-compatibility-version none -wmo -Xfrontend -disable-objc-interop) | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: optimized_stdlib
+// REQUIRES: VENDOR=apple
+// REQUIRES: OS=macosx
```

Comment:
I keep forgetting whether we still need `REQUIRES: asserts` when `-enable-experimental-feature` flag is used or not, could you please double-check?

Reply:
Some features need asserts, some don't (it's controlled in the Features.def file on a per feature basis). The ones used here don't need asserts.